main.js

```diff
@@ -16,108 +16,145 @@ let bootstrapFixtures = {
   worldLinkLinguageBundle:  fs.readFileSync(path.join('./bootstrap/', worldLinkLanguageHash, 'bundle.js')),
   worldLinkLinguageMeta: JSON.parse(fs.readFileSync(path.join('./bootstrap/', worldLinkLanguageHash, 'meta.json'))),
 }
+const {ipcMain} = require('electron');
+const { exit } = require("process");
+
+let spawnExecutor = false
+console.log(process.argv)
+if(process.argv.length === 4 && process.argv[2] === "executor") {
```

Comment:
Do we need to pass executor as command-line argument? We could check if the app is in production using `app.isPackaged`.